docs/policies/support.md

```diff
@@ -1,69 +1,69 @@
-   after which the releases will only address the most critical bug fixes and security fixes for at least another 12 months.
+3. **Maintenance** - 일반적으로 유지보수 모드는 다음 주 버전이 Active로 전환되는 동시에 발표되며,
+   그 이후에는 최소 12개월 동안 가장 중요한 버그 수정 및 보안 수정만 처리합니다.
 
-4. **Community** - SDK will no longer receive updates from Microsoft unless otherwise specified in the separate customer agreement.
-   The package will remain available via public package managers and the GitHub repo, which can be maintained by the community.
+4. **Community** - SDK는 별도의 고객 계약에 달리 명시되지 않는 한 더 이상 Microsoft로부터 업데이트를 받지 않습니다.
+   패키지는 커뮤니티가 관리할 수 있는 공용 패키지 관리자와 GitHub repo를 통해 계속 사용할 수 있습니다.
 
-You can check the lifecycle stage for your package at [this page](https://azure.github.io/azure-sdk/releases/latest/index.html)
+[해당 페이지](https://azure.github.io/azure-sdk/releases/latest/index.html) 에서 패키지의 수명 주기 단계를 확인할 수 있습니다.
 
-### **Azure Clouds**
+### **Azure 클라우드**
 
-By default, the Azure libraries are configured to connect to the Global Azure Cloud.
-Additional cloud target platforms are available, such as Azure Stack, Azure China and Government Cloud.
-Package lifecycle may vary across different target platforms. Refer to the target platform documentation for more information.
+기본적으로 Azure 라이브러리는 글로벌 Azure 클라우드에 연결하도록 구성됩니다.
+Azure Stack, Azure China 또는 Government 클라우드와 같은 추가 클라우드 플랫폼을 사용할 수 있습니다.
+패키지 수명 주기는 대상 플랫폼에 따라 다를 수 있습니다. 자세한 내용은 대상 플랫폼 설명서를 참조하십시오.
 
-### **Azure SDK dependencies**
+### **Azure SDK 종속성**
 
-The Azure SDK libraries depend on Azure Service REST API, programming language runtime, OS, and third-party libraries.
+Azure SDK 라이브러리는 Azure Service REST API, 프로그래밍 언어 런타임, OS 및 타사 라이브러리에 따라 달라집니다.
 
-> The Azure SDK libraries will not be guaranteed to work on platforms and with other dependencies that have reached their end of life. Dropping support for such dependencies may be done without increasing the major version of the Azure SDK libraries. We strongly recommend migration to supported platforms and other dependencies to be eligible for technical support.
+> Azure SDK 라이브러리는 수명이 다한 플랫폼 및 기타 종속성에서는 작동을 보장하지 않습니다. Azure SDK 라이브러리의 주 버전을 늘리지 않고 이러한 종속성에 대한 지원을 중단할 수 있습니다. 기술 지원을 받으려면 지원되는 플랫폼 및 기타 종속성으로 마이그레이션하는 것을 권장드립니다.
 
-Below is a list of supported Azure SDK platforms and runtimes for your reference:
+다음은 참조용으로 지원되는 Azure SDK 플랫폼 및 런타임 목록입니다:
 
-**Operating Systems:** Windows 10, macOS-10.15 , Linux (tested on Ubuntu 18.04)
-For Mobile development, please check the [IOS supported platforms](https://azure.github.io/azure-sdk/ios_design.html#ios-library-support), and the [Android supported platforms](https://azure.github.io/azure-sdk/android_design.html)
+**운영체제:** Windows 10, macOS-10.15 , Linux (Ubuntu 18.04에서 테스트됨)
+모바일 개발의 경우 [IOS 지원 플랫폼](https://azure.github.io/azure-sdk-korean/ios_design.html#ios-library-support), [Android 지원 플랫폼](https://azure.github.io/azure-sdk-korean/android_design.html)을 확인해주세요.
```

Comment:
Since the two target links are not translated to Korean, I wanna suggest to keep using English links